tools/evaluate_queries/def.hpp

```diff
@@ -0,0 +1,21 @@
+#include "block_freq_index.hpp"
+#include "cursor/scored_cursor.hpp"
+#include "freq_index.hpp"
+#include "index_types.hpp"
+#include "query/queries.hpp"
+#include "scorer/bm25.hpp"
+#include "scorer/dph.hpp"
+#include "scorer/pl2.hpp"
+#include "scorer/qld.hpp"
+
+namespace pisa {
+
```

Comment:
I guess `def.hpp` stays for defines. Shall we rename it to `defines.hpp` or something similar?

Reply:
In that case, I would just go with `definitions.hpp`